intlogger.go

```diff
@@ -635,8 +635,21 @@ func (l *intLogger) StandardLogger(opts *StandardLoggerOptions) *log.Logger {
 }
 
 func (l *intLogger) StandardWriter(opts *StandardLoggerOptions) io.Writer {
+        newLog := &intLogger{
+		json:              l.json,
+		name:              l.name,
+		timeFormat:        l.timeFormat,
+		mutex:             l.mutex,
+		writer:            l.writer,
+		level:             l.level,
+		exclude:           l.exclude,
+		independentLevels: l.independentLevels,
+	}
+	if l.callerOffset > 0 {
+		newLog.callerOffset = l.callerOffset + 4
```

Comment:
Why is this 4? How did you derive 4?

Reply:
when we call Go log.go (logger).printf,
the stack is 
logger.printf() -> l.Output() ->l.out.writer(hclog:stdlogAdaptor.write) -> hclog:stdlogAdaptor.dispatch()
So is 4.

Reply:
Ok, can you please add a comment about that here?

Reply:
By here, I mean in the code.

Reply:
done